spec/rails_helper.rb

```diff
@@ -65,6 +65,10 @@
   # config.filter_gems_from_backtrace("gem name")
 
   # Include integration helpers for feature tests
-  config.include(Devise::Test::IntegrationHelpers, type: :feature)
+  config.include(Devise::Test::IntegrationHelpers, type: :system)
   config.include(Devise::Test::ControllerHelpers, type: :controller)
+
+  config.before(:each, type: :system) do
+    driven_by(:selenium_chrome_headless)
+  end
```

Comment:
In [spec/support/capybara.rb](https://github.com/abtion/muffi/blob/main/spec/support/capybara.rb#L13) we also set the webdriver (and allow it to be overridden by an env var).

I wonder why these new three lines are needed? If they override what we have in `capybara.rb`?

Reply:
I am uncertain, we should try it without those lines.
And if they are needed for some reason, they should use the same env var and be moved to `capybara.rb`.

Reply:
We add this so it runs headlessly (does not open chrome)

Reply:
Yes, but the same thing is configured in `capybara.rb`.
Either way, I think it should be moved to `capybara.rb`, since it is Capybara specific configuration.